README.md

```diff
@@ -1,80 +1,118 @@
-[![Build Status](https://travis-ci.org/weldr/bdcs.svg?branch=master)](https://travis-ci.org/weldr/bdcs)
-[![Coverage Status](https://coveralls.io/repos/github/weldr/bdcs/badge.svg?branch=master)](https://coveralls.io/github/weldr/bdcs?branch=master)
+The bdcs project is the main backend component of welder.  It is responsible for
```

Comment:
Are we welder or weldr or Weldr? I'm currently not 100% sure, but think it should match our github name.

Reply:
I don't know either, but fixed to match the github name.  It's easy enough to change later.